tools/cli/commands/changelog.js

```diff
@@ -369,7 +370,8 @@ async function changelogAdd( argv ) {
 
 	for ( const proj of uniqueProjects ) {
 		console.log( chalk.green( `Running changelogger for ${ proj }!` ) );
-		const response = await promptChangelog( argv, proj, projectChangeTypes[ proj ] );
+		const projArray = [ proj ];
+		const response = await promptChangelog( argv, projArray, projectChangeTypes[ proj ] );
```

Comment:
Does it not work to just do like this?
```suggestion
		const response = await promptChangelog( argv, [ proj ], projectChangeTypes[ proj ] );
```